arrow-fx-coroutines-test/src/main/kotlin/arrow/fx/coroutines/predef-test.kt

```diff
@@ -154,6 +154,9 @@ fun <O> Arb.Companion.function(arb: Arb<O>): Arb<() -> O> =
 fun Arb.Companion.unit(): Arb<Unit> =
   Arb.constant(Unit)
 
+fun <A, B> Arb.Companion.functionAToB(arb: Arb<B>): Arb<(A) -> B> = arb.map { b: B -> { _: A -> b } }
+fun <A> Arb.Companion.nullable(arb: Arb<A>): Arb<A?> = Arb.Companion.choice(arb, arb.map { null })
```

Comment:
Small formatting nit 😁

```suggestion
fun <A, B> Arb.Companion.functionAToB(arb: Arb<B>): Arb<(A) -> B> =
  arb.map { b: B -> { _: A -> b } }
  
fun <A> Arb.Companion.nullable(arb: Arb<A>): Arb<A?> =
  Arb.Companion.choice(arb, arb.map { null })
```